src/hotspot/share/gc/g1/g1EvacuationFailureRegions.cpp

```diff
@@ -0,0 +1,84 @@
+/*
```

Comment:
It's a bit weird that all other similar files start with "g1EvacFailure" and this one with "g1EvacuationFailure" - please change according to existing style (or rename all others first in a separate CR).

Reply:
Sure. I created JDK-8273218 to modify just g1EvacuationInfo.hpp and related class name, as it's more simple than modify every EvacXxx to EvacuationXxx.